common/evaluate.py

```diff
@@ -7,9 +7,10 @@ class EvaluatorFactory(object):
     Get the corresponding Evaluator class for a particular dataset.
     """
     evaluator_map = {
+        'ApacheDiffToken': DiffTokenEvaluator,
+        'SpringDiffToken': DiffTokenEvaluator,
         'VulasDiffToken': DiffTokenEvaluator,
         'VulasPairedToken': PairedTokenEvaluator
```

Comment:
For now we are only using the 3 DiffToken datasets, right? i.e., the vulasPairdToken is not used?

Reply:
We do use that, and it is current